dashboards/azure-custom-vision-training/azure-custom-vision-training.json

```diff
@@ -0,0 +1,350 @@
+
+  "pages": [
+    {
+      "name": "Cognitive Service Custom Vision Training",
+      "description": null,
+      "widgets": [
+        {
+          "title": "Total calls",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 3,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT SUM(`azure.cognitiveservices.accounts.TotalCalls`) AS 'Total Calls' WHERE collector.name='azure-monitor' AND azure.resourceType='microsoft.cognitiveservices/accounts' AND (azure.cognitiveservices.accounts.apiname LIKE '%CustomVision%' OR azure.cognitiveservices.accounts.apiname LIKE '%Custom_Vision_Training%')"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "WARNING"
+              }
+            ]
+          }
+        },
+        {
+          "title": "Total errors",
+          "layout": {
+            "column": 4,
+            "row": 1,
+            "width": 3,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.billboard"
+          },
+          "rawConfiguration": {
+            "facet": {
+              "showOtherSeries": false
+            },
+            "nrqlQueries": [
+              {
+                "accountIds": [],
+                "query": "FROM Metric SELECT SUM(`azure.cognitiveservices.accounts.TotalErrors`) AS 'Total Errors' WHERE collector.name='azure-monitor' AND azure.resourceType='microsoft.cognitiveservices/accounts' AND (azure.cognitiveservices.accounts.apiname LIKE '%CustomVision%' OR azure.cognitiveservices.accounts.apiname LIKE '%Custom_Vision_Training%')"
+              }
+            ],
+            "platformOptions": {
+              "ignoreTimeRange": false
+            },
+            "thresholds": [
+              {
+                "alertSeverity": "CRITICAL",
+                "value": 0
+              }
+            ]
+          }
+        },
+        {
+          "title": "",
+          "layout": {
+            "column": 7,
+            "row": 1,
+            "width": 6,
+            "height": 3
+          },
+          "linkedEntityGuids": null,
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "# Azure Custom Vision Training\nAzure Custom Vision is an image recognition service that lets you build, deploy, and improve your own image identifier models. An image identifier applies labels to images, according to their visual characteristics."
```

Comment:
Same thing here, could it be moved to the top left corner?

Reply:
same as above